README.md

```diff
@@ -9,16 +9,16 @@ A developer would use the core connect-sdk package in conjunction with 1 or more
 Getting started is simple, just import and pass in the contexts to the Wormhole class.
 
 ```ts
-import { Wormhole, Context, Network } from '@wormhole-foundation/connect-sdk';
+import { Wormhole, Context } from '@wormhole-foundation/connect-sdk';
 import { EvmContext } from '@wormhole-foundation/connect-sdk-evm';
 import { SolanaContext } from '@wormhole-foundation/connect-sdk-solana';
 
-const NETWORK = Network.MAINNET;
+const network = "Mainnet";
```

Comment:
I def prefer the const.  Because you can mess up a string pretty easily, like it can be hard to remember if its "Mainnet", "mainnet" or "MAINNET"

Reply:
Imho the main (and really only) advantage of the const is that it stands out as being a const over a normal string.

However, I think that the disadvantage of the consts strongly outweigh that:

* Definitions become more awkward:
Take the example of networks:
`export const networks = ["Mainnet", "Testnet", "Devnet"] as const;
export type Network = (typeof networks)[number];
export const isNetwork = (network: string): network is Network =>
  networks.includes(network as Network);`
If you want to declare consts you will have to repeat yourself in so way:
either:
`const MAINNET = "Mainnet";
const TESTNET = "Testnet";
const DEVNET = "Devnet";
const networks = [MAINNET, TESTNET, DEVNET];`

or the other way around by using a destructuring assignment.

And this is just the tip of the iceberg:
Do we introduce constants for all chain names?
If yes: Is SOLANA now defined in chains or in platforms?
If no: That's oddly inconsistent.

* Whenever a type is expected you see the definition of the type e.g. if you have a function `doSomething(network: Network)` your editor will tell you that the 3 valid values are the 3 string literals. But if we have constants, instead of using the straight forward string literals, you are supposed to import and use a constant instead even though nothing in the code tells you to. So even if we are conscientious with using the constants in the sdk itself you're all but guaranteed to have user code that will simply use the string literals anyway.

* There is no risk of typos or spelling confusion since the Typescript will enforce that you are passing a correct type literal and instead of being all capitalized it's simply pascal case which is easier to type anyway (especially if - like me - you have no capslock on your keyboard)

Reply:
Ok yeah, good points, let's go with the strings 👌

Reply:
same